content/ways-to-train/_other_ways_to_train.md

```diff
@@ -1,3 +1,19 @@
 ## Other ways to train
 
-There are a number of specialised routes into teaching. For more information about ways to train, [read more detailed guidance](/guidance/become-a-teacher-in-england) or talk to us to discuss your options.  
\ No newline at end of file
+There are a number of specialised routes into teaching. 
+
+### Career changers
+
+If you want a [career change and fancy becoming a teacher](/guidance/become-a-teacher-in-england#career-changers), you’ll get the chance to use your life and work experience coupled with your passion for your subject to inspire young people. There's plenty of support to get into teaching and make the career change.
+
+### Researchers in schools
+
+You’ll need (or be about to finish) a doctorate in the [subject you want to teach](/guidance/become-a-teacher-in-england#researchers-in-schools-candidates-with-a-doctorate). You’ll then train over 3 years with a bursary. If you have a lot of relevant work experience you could train and earn a salary.
```

Comment:
I suspect "you could train and earn a salary" will garner some attention, could this link to somewhere that details how?

Reply:
Good shout - done.